usr/lib/linuxmuster-webui/plugins/lmn_linbo_sync/api.py

```diff
@@ -143,19 +143,21 @@ def list_workstations(context):
         'lmnReserved14',
         'sophomorixComment',
         'options',
-    ]    
+    ]
     with LMNFile(path, 'r', fieldnames=fieldnames) as devices_csv:
 
         devices = devices_csv.read()
         for device in devices:
+            if school != 'default-school':
+                if device['hostname']:
+                    device['hostname'] = school + '-' + device['hostname']
```

Comment:
I'm converting all string concatenation to f-strings, like you did below in the quota, for more readability : 

```python
device['hostname'] = f'{school}-{device["hostname"]}'
```

Reply:
Yes I do this for myself for a while but picked concatenation for continuity reasons in this case. I'll update it to f-string.